.github/workflows/package-apisix-dashboard-rpm-ubi.yml

```diff
@@ -0,0 +1,75 @@
+name: package apisix-dashabord rpm for ubi
```

Comment:
We don't need to build an ubi package for the dashboard. 
This file can be deleted.

Reply:
Done deleted 👍